specification/dimensions/skupriceid.md

```diff
@@ -0,0 +1,30 @@
+# SKU Price ID
+
+A SKU Price ID is a unique identifier that defines the unit price used to calculate the charge. SKU Price ID can be referenced on a price list published by a provider to look up detailed information, including a corresponding list unit price. The composition of the properties associated with the SKU Price ID may differ across providers. SKU Price ID is commonly used for analyzing cost based on pricing properties such as Terms and Tiers.
+
+The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The ListUnitPrice MUST be associated with the SKUPriceID in the provider published pricing sheet. The SkuPriceId MUST NOT be null when ChargeType is ‘Purchase’ or ‘Usage’. The SkuPriceId SHOULD NOT be null when ChargeSubcategory is ‘Refund’.
```

Comment:
Did we standardize on "price list"? (I'm not 100% sure.)
Quotes for consistency.
Switching the SHOULD to a MUST in line with ChargeSubcategory.
```suggestion
The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The ListUnitPrice MUST be associated with the SKUPriceID in the provider published price list. The SkuPriceId MUST NOT be null when ChargeType is "Purchase" or "Usage". The SkuPriceId MUST NOT be null when ChargeSubcategory is "Refund" and the refund is related to charges with a specific SkuPriceId.
```